### ISSUE TITLE
Search features

### ISSUE THREAD

**Issue by abhijeetxenonstack** [tailwarden/komiser#633]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
![Screenshot from 2023-03-16 18-59-45](https://user-images.githubusercontent.com/124243238/225632366-61f5c4b6-7dd6-433b-bd01-c89e923c843b.png)
 
i want to know that can a user add more resource options according to their requirement in resource option if yes then guide me how ?

**Comment by aimanafzal** [comment]
@jakepage91 What this feature actually means? Can this be elaborated?

**Comment by jakepage91** [comment]
Yeah, I'm not sure which resource options @abhijeetxenonstack is requesting to be added. I we are going to have to wait for him to get back to us @aimanafzal

**Comment by ShubhamPalriwala** [comment]
Hey @abhijeetxenonstack could you help us with exactly you are looking for? We can try get it on our roadmap and shipped for your easier usability! 🚀
